include/cuda/std/detail/libcxx/include/__cuda/climits_prelude.h

```diff
@@ -15,11 +15,11 @@
 #error "<__cuda/climits_prelude> should only be included in from <cuda/std/climits>"
 #endif // __cuda_std__
 
-#ifndef __CUDACC_RTC__
+#ifndef _LIBCUDACXX_COMPILER_NVRTC
     #include <climits>
     #include <limits.h>
     #include <cstdint>
-#else
+#else // ^^^ !_LIBCUDACXX_COMPILER_NVRTC ^^^ / vvv _LIBCUDACXX_COMPILER_NVRTC vvv
```

Comment:
I wonder if we can lint this across the code base. It would make the config much more fun to read.